src/components/Chat/Playback/PlaybackAttachments.tsx

```diff
@@ -0,0 +1,41 @@
+import { IconFile, IconX } from '@tabler/icons-react';
+
+import { PlaybackAttachmentType } from './playback.types';
+
+interface PlaybackAttachmentProps {
+  attachmentTitle: string;
+}
+
+interface PlaybackAttachmentsProps {
+  attachments: PlaybackAttachmentType[];
+}
+
+export function PlaybackAttachment({
```

Comment:
Is it impossible to use just ChatAttachment? It will be better, because if styles will change we will not need to do it in multiple places

Reply:
Adjusted to use ChatInputAttachments here.